daemon/delete_test.go

```diff
@@ -23,68 +23,61 @@ func newDaemonWithTmpRoot(t *testing.T) (*Daemon, func()) {
 	return d, func() { os.RemoveAll(tmp) }
 }
 
-// TestContainerDeletePaused tests that a useful error message and instructions is given when attempting
-// to remove a paused container (#30842)
-func TestContainerDeletePaused(t *testing.T) {
-	c := &container.Container{
+func newContainerWithIDAndConfig(state *container.State) *container.Container {
```

Comment:
The function name feels misleading to me. Neither the ID nor the config are supplied to this function. `newContainerWithState`?